FRCore/FRCore/Keychain/EC2PrimeRandomKey.swift

```diff
@@ -107,7 +107,7 @@ public struct ECPrimeRandomKey {
         //  Public Key Query
         var publicQuery: [String: Any] = [:]
         publicQuery[String(kSecAttrLabel)] = publicKeyLabel
-        publicQuery[String(kSecAttrAccessControl)] = SecAccessControlCreateWithFlags(kCFAllocatorDefault, kSecAttrAccessibleAlwaysThisDeviceOnly, [], nil)!
+        publicQuery[String(kSecAttrAccessControl)] = SecAccessControlCreateWithFlags(kCFAllocatorDefault, kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly, [], nil)!
```

Comment:
Have you checked if that breaks any notification or extension stuff? I know we don't do officially extensions but I recall a use case that an extension was used, it might be the authenticator module. As a general idea I like the `kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly` of course as long as we don't break anything unexpectedly

Reply:
I reviewed this with @george-bafaloukas-forgerock. The one I'm using is the most appropriate one as the older one is deprecated. However we need to test this during the release for Push Notifications.